releaseNotes/JavaScriptProvider.md

```diff
@@ -0,0 +1,33 @@
+# Azure App Configuration JavaScript Provider
+
+[Source code][source_code] | [Package (npm)][package] | [Samples][samples]
+
+## 1.0.0-preview.2 - December 15, 2023
+## Breaking Changes
+- Changed the behavior when multiple labels are filtered in one selector. In the JavaScript provider configuration settings are loaded into a `Map` containing `key` and `value`, without preserving the `label`. If multiple labels are matched in one selector, the provider cannot determine which one to use. Now it will throw an error if label filter contains `*` or `,`. [#22](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/22)
+
+## Enhancements
+- Added more information to the doc comments for better IntelliSense. [#32](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/32)

```

Comment:
Doesn't sound like something worth calling out in release notes.

Reply:
removed.